### PR TITLE
Drop support for CUDA 8.0 and 9.1

### LINKED ISSUE #3301: Drop support for CUDA 8.0 and 9.1 in v8
As users of CUDA 8.0 and 9.1 are decreasing, we are planning to drop support for these CUDA versions.

### PR DESCRIPTION
**Pull request by kmaehashi**
Fix #3301

### CODE REVIEW

File: docs/source/install.rst [hunk 1]
```diff
@@ -53,9 +53,9 @@ Optional Libraries
 Some features in CuPy will only be enabled if the corresponding libraries are installed.
 
 * `cuDNN <https://developer.nvidia.com/cudnn>`_ (library to accelerate deep neural network computations)
-    * Supported Versions: v5, v5.1, v6, v7, v7.1, v7.2, v7.3, v7.4 and v7.5.
+    * Supported Versions: v7, v7.1, v7.2, v7.3, v7.4 and v7.5.
```
Comment (kmaehashi) [inline docs/source/install.rst:56]: The minimum version of cuDNN is v7.0.4.

File: docs/source/install.rst [hunk 2]
```diff
@@ -53,9 +53,9 @@ Optional Libraries
...
 * `NCCL <https://developer.nvidia.com/nccl>`_  (library to perform collective multi-GPU / multi-node computations)
-    * Supported Versions: v1.3.4, v2, v2.1, v2.2, v2.3 and v2.4.
+    * Supported Versions: v2, v2.1, v2.2, v2.3 and v2.4.
```
Comment (kmaehashi) [inline docs/source/install.rst:58]: The minimum version of NCCL that run on CUDA 9.0 is v2.

**Comment by emcastillo** [comment]
Jenkins, test this please

File: docs/source/install.rst (hunk 2 above)
Comment (leofang) [reply docs/source/install.rst:58]: Great! We can remove a lot of compatibility code for NCCL v1.x later!

**Pull request merged**